base/util.jl

```diff
@@ -243,9 +243,6 @@ function julia_cmd(julia=joinpath(Sys.BINDIR, julia_exename()); cpu_target::Unio
     end
     if opts.use_pkgimages == 0
         push!(addflags, "--pkgimages=no")
-    else
-        # If pkgimage is set, malloc_log and code_coverage should not
-        @assert opts.malloc_log == 0 && opts.code_coverage == 0
```

Comment:
Maybe we should check that malloc_log/code_coverage don't reach the `generating-ouput` process?